{
  "modified": "2024-12-09T06:50:34Z",
  "published": "2024-12-09T06:50:34Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-11630",
  "summary": "Malicious code in matlibplot (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "matlibplot",
        "purl": "pkg:pypi/matlibplot"
      },
      "versions": [
        "0.3"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "id": "RLMA-2024-11085",
        "import_time": "2024-12-09T14:38:45.46227961Z",
        "modified_time": "2024-12-09T06:50:34Z",
        "sha256": "9f0477f858af87764b6e9c7486403bbfc93331b47027f3616f37c2390c2f74c8",
        "source": "reversing-labs",
        "versions": [
          "0.3"
        ]
      }
    ]
  }
}
